src/libguac/user-handshake.c

```diff
@@ -290,7 +234,11 @@ int guac_user_handle_connection(guac_user* user, int usec_timeout) {
 
     guac_socket* socket = user->socket;
     guac_client* client = user->client;
-
+    
+    user->info.audio_mimetypes = NULL;
+    user->info.image_mimetypes = NULL;
+    user->info.video_mimetypes = NULL;
```

Comment:
What about `user->info.timezone`?

Reply:
We can definitely do that, here.  It doesn't break things to not have it, here, because the `free()` call is just a standard call rather than a loop that goes through a pointer array, as in the case of the audio, image, and video mimetypes.  But it's trivial to initialize it, so I'll add that.

Reply:
Added a `NULL` assignment for `user->info.timezone`, here, to match the others.